scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -95,50 +96,44 @@ public void initialize() {
       // set the slider to display tick labels as time strings
       setSliderLabels();
 
-      
+      // set the combo box to display the lighting modes as strings
+      setComboBoxLabels();
```

Comment:
I think you might not need this any more if you're using the ListCell method to modify the ComboBox display of each entry